### DISCUSSION TITLE
How does staggering works?

### DISCUSSION THREAD

**Discussion by jacquesyeoh** [swar/Swar-Chia-Plot-Manager#133] [Q&A]
Let's say there are two jobs, Job A with stagger 40 minutes and Job B with 0 stagger
However, when manager.py starts, both job started.

Does it mean
1. All jobs will start without delay even with stagger settings ON
2. stagger settings only apply after 2nd run onwards?

Is there a sample configuration for common setups such as 16T with 2x1TB SSD?

  - name: A
    max_plots: 999
    temporary_directory: G:\Cache
    temporary2_directory:
    destination_directory:
      - D:\Plot2
      - D:\Plot3
      - D:\Plot4
    size: 32
    bitfield: true
    threads: 4
    buckets: 128
    memory_buffer: 3408
    max_concurrent: 1
    max_concurrent_with_start_early: 2
    stagger_minutes: 40
    max_for_phase_1: 1
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false

  - name: B
    max_plots: 999
    temporary_directory: G:\Cache
    temporary2_directory:
    destination_directory:
      - K:\Plot2
      - K:\Plot3
      - K:\Plot4
    size: 32
    bitfield: true
    threads: 4
    buckets: 128
    memory_buffer: 3408
    max_concurrent: 3
    max_concurrent_with_start_early: 4
    stagger_minutes: 0
    max_for_phase_1: 2
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false

**Comment by Berni-k** [comment]
I am having the same issue. 

I plot on 4 seperate SSDs in parallel, but this tool keep starting 4 plotting jobs simultaneously. This causes problems later on because they finish simultaneously, so many plots start getting copied simultaneously over to my NAS and clogging things up.

EDIT: Looked trough the source code and it looks like its not possible. So i added this feature myself, it adds a extra config setting for the global number of minutes between starting new plotting jobs. Will make a git pull with it later if anyone is interested

**Reply by alexhu98** [reply]
Thanks for the code code. Please make a git pull.
Another use has also request it in [delay between jobs #360](https://github.com/swar/Swar-Chia-Plot-Manager/discussions/360)

**Reply by Berni-k** [reply]
No problem, sent in pull https://github.com/swar/Swar-Chia-Plot-Manager/pull/380
